symplyphysics/laws/astronomy/eccentricity_of_orbit.py

```diff
@@ -0,0 +1,42 @@
+from sympy import (Eq, solve, sqrt, S)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+    dimensionless,
+)
+
+# Description
+## Many bodies in space move in elliptical orbits. For an elliptical orbit, the eccentricity can be calculated.
+## The eccentricity can be used to judge the elongation of the elliptical orbit.
+
+## Law is: e = sqrt(1 - (b / a)^2), where
+## e - eccentricity  of orbit,
+## b - small semi-axis of ellipse,
+## a - large semi-axis of ellipse.
+
+eccentricity = Symbol("eccentricity", dimensionless)
+
+small_semi_axis = Symbol("small_semi_axis", units.length)
+large_semi_axis = Symbol("large_semi_axis", units.length)
+
+law = Eq(eccentricity, sqrt(1 - (small_semi_axis / large_semi_axis)**2))
+
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(small_semi_axis_=small_semi_axis, large_semi_axis_=large_semi_axis)
+@validate_output(eccentricity)
+def calculate_eccentricity(small_semi_axis_: Quantity, large_semi_axis_: Quantity) -> float:
+    result_expr = solve(law, eccentricity, dict=True)[0][eccentricity]
```

Comment:
Validate that small axis is smaller than large axis. Or select them automatically.